### ISSUE TITLE
Test video is so bad!

### ISSUE THREAD

**Issue by Jason-xin** [AntiAegis/Human-Segmentation-PyTorch#5]
![image](https://user-images.githubusercontent.com/13817995/61289191-5cf99080-a7fb-11e9-8ed6-2333201939e6.png)
![image](https://user-images.githubusercontent.com/13817995/61289198-6256db00-a7fb-11e9-9faf-52ea609cf0da.png)
![image](https://user-images.githubusercontent.com/13817995/61289210-65ea6200-a7fb-11e9-9463-5de859a96d6f.png)

**Comment by NguyenDangBinh** [comment]
dear,
fps is so low? it only gain 1.8?

**Comment by Jason-xin** [comment]
> dear,
> fps is so low? it only gain 1.8?

Yes, I just test it on my Mac

**Comment by thuyngch** [comment]
* The speed is dependent on your hardware.
* Regarding the accuracy, this repository is merely a benchmark on two small datasets. However, you can use this repository to retrain models on your larger datasets for gaining higher performance in term of accuracy.

**Comment by SouthMemory** [comment]
So bad, should consider not to provide pretrained model 

![frame_《焦点访谈》20200829北影节：一场云上盛宴__000155_0](https://user-images.githubusercontent.com/28387896/102449659-bceb0b00-406f-11eb-9f35-278fb3506eae.png)
